kolibri/plugins/management/assets/src/vue/user-page/user-create-modal.vue

```diff
@@ -55,22 +55,25 @@
     },
     data() {
       return {
-        username: '',
-        password: '',
-        firstName: '',
-        lastName: '',
+        user: {
+          username: '',
+          password: '',
+          full_name: '',
+        },
         role: 'learner',
       };
     },
     methods: {
       createNewUser() {
-        const payload = {
-          password: this.password,
-          username: this.username,
-          full_name: this.fullName,
-          facility: this.facility,
-        };
-        this.createUser(payload, this.role);
+        this.user.facility = this.facility;
```

Comment:
this doesn't really make sense because `this.user.facility` isn't a data attribute. Iterating over the `getOwnPropertyNames` is kind of a code smell because future bugs can be introduced if variable types change.

Recommendation for future refactor: https://trello.com/c/1YRkWjHQ/387-make-view-model-data-access-explicit-rather-than-iterating-over-attribute-names